index.js

```diff
@@ -65,8 +65,42 @@ function toggleColor(element) {
     } else {
         element.style.color = "green"
     }
-}
+} 
+
+header.addEventListener('click', function(event){
+    toggleColor(event.target)
+})
```

Comment:
great work! We have the header variable already defined earlier in the file.
All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.

Great job incorporating the `toggleColor` function and passing in the event target as an argument to the function.

```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})

```

With ES6 notation

```
header.addEventListener("click", e => toggleColor(e.target))
```